packages/@coorpacademy-components/src/types/progression-engine.d.ts

```diff
@@ -321,7 +321,7 @@ export type AcceptedAnswers = Array<Answer>;
 
 export type ChoiceItem = {
   text: string;
-  value: string;
+  selected: boolean;
```

Comment:
sûr ? ce sont des types de progression-engine... 🤔

Reply:
je remettre une value optionelle alors,

- effectivement on "imite" le type d'un `choice.items[i]` d'ici https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-progression-engine/src/types.js#L289

- mais coté frontend/redux on a bien besoin de ce `selected`

il faudrait un `ChoiceItemAPI` étendu par un `ChoiceItem` .. je ne suis pas très fan, mais je n'ai pas un avis tranché la dessus, tu en penses quoi ?

Reply:
ok noté, je le prends à la suite sur une autre PR https://github.com/CoorpAcademy/components/issues/2381